### ISSUE TITLE
About Readme.md

### ISSUE THREAD

**Issue by koba-jon** [basiclab/GNGAN-PyTorch#1]
Hello,
I found a very interesting technique.

There may be a mistake about "How to integrate Gradient Normalization into your work?" of Readme.md.

Now it's the following:
```
loss_real = loss_fn(pred_real, torch.ones_like(pred_real))
loss_fake = loss_fn(pred_fake, torch.ones_like(pred_fake))
```

Is the following correct?
```
loss_real = loss_fn(pred_real, torch.ones_like(pred_real))
loss_fake = loss_fn(pred_fake, torch.zeros_like(pred_fake))
```
The `losses.py` in your repository was exactly like this.

Thank you.

**Comment by w86763777** [comment]
Thanks for your reminding, mistakes have been corrected.
